src/cms/database/migrations/2025.09.05T00.00.24.park-area.js

```diff
@@ -0,0 +1,2082 @@
+"use strict";
+
+const rows = [
+  {
+    orcs: "1",
+    parkAreaName: "Buttle Lake Campground",
+    parkAreaType: "Frontcountry Campground",
+    orcsAreaNumber: "1-1",
+  },
```

Comment:
This `orcsAreaNumber` is a combination of `orcs` and `area_id`? or `orcs` and `index`?
Just wanted to make sure because `featureId` in the `park-operation-sub-area` uses `orcs` and `feature_id`.